app/presenters/education_navigation_ab_testable.rb

```diff
@@ -0,0 +1,17 @@
+module EducationNavigationABTestable
```

Comment:
@alecgibson cool - although if you're keeping the module implementation a better place still would be `app/controllers/concerns`. It's quite similar to other modules in there already.

Otherwise fine with this, given we're nuking it all after the AB test's finished.